tests/integration/tls_provider_charm/charmcraft.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: "charm"
+bases:
+  - build-on:
+      - name: "ubuntu"
+        channel: "22.04"
```

Comment:
Any reason not to start with 24.04 for this?

Reply:
Updated